app/templates/suppliers_details.html

```diff
@@ -55,14 +55,26 @@
         <dl class="visually-hidden">
           <dt>Capabilities</dt>
         </dl>
-        {% for category in supplier_categories|sort() %}
+        {% for category in supplier["categories"]|sort() %}
         <dd>
           <a href="#" class="disabled">{{ category }}</a>
         </dd>
         {% endfor %}
       </footer>
     </article>
   </li>
+  {% if supplier["caseStudies"]|count > 0 %}
+  <li>
+    <p>Case Studies</p>
+    <article>
+      {% for casestudy in supplier["caseStudies"] %}
+        {% with component = render_component('bundles/CaseStudy/CaseStudyViewWidget.js',{ "casestudy": casestudy }) %}
```

Comment:
Added jinja previews that link to the full view to match designs.
![](https://cloud.githubusercontent.com/assets/81432/19503089/00de0334-95fd-11e6-92ff-c4254382f0eb.png)